src/Service/LearningMaterialTextExtractor.php

```diff
@@ -59,7 +60,9 @@ public function extract(LearningMaterialDTO $dto): void
             $this->iliosFileSystem->storeLearningMaterialText($dto->relativePath, $text);
         } catch (Exception $exception) {
             if (
-                $exception->getCode() === 422 &&
+                // error code from php-tika library (422)
+                // https://github.com/vaites/php-apache-tika/blob/792dc1254b4ccd92c6964ab477a1626dca6784ee/src/Clients/WebClient.php#L620
```

Comment:
appreciate the additional code comment and link.
also, lol sonarqube "the line is too long" :clown_face:

Reply:
What I'm really happy with is that apparently phpcs has a way to know that it's a comment that can't be broken into another line. Though I wish GitHub hadn't axed their link shorter, that would have been useful here.